src/feature/wines/components/card/DetailCard.tsx

```
@@ -1,7 +1,7 @@
 import Image from 'next/image';
 
 import CardDropdownMenu from '@/feature/wines/components/button/CardDropdownMenu';
-import { WineDetailResponse } from '@/feature/wines/schema/wine.schema';
+import { GetWineDetailResponse } from '@/feature/wines/schema/wine.schema';
```

ESLint

[error] 'GetWineDetailResponse' is defined but never used.

(@typescript-eslint/no-unused-vars)